adagrams/game.py

```diff
@@ -1,13 +1,79 @@
 from random import randint
 
+LETTER_POOL = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2, 'I': 9, 'J': 1, 'K': 1, 'L': 4, 
+    'M': 2, 'N': 6, 'O': 8, 'P': 2, 'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1, 
+    'Y': 2, 'Z': 1
+}
+
 def draw_letters():
-    pass
+    letter_list = []
+    hand = []
+
+    for letter in LETTER_POOL:
+        letter_list += [letter] * LETTER_POOL[letter]
+
+    for i in range(10):
+        random_num = randint(0, len(letter_list) - 1)
```

Comment:
It's true that the return value of executing `randint` is a random number, but what does that random number signify in your code?

It's a random index you'll use to access a random element from `letter_list`. I'd prefer a more descriptive name like `random_index` to make your code more self-documenting. 
```suggestion
        random_index = randint(0, len(letter_list) - 1)
```